### PR TITLE
Update Move-CsUser.md

### PR DESCRIPTION
**Pull request by ruchiagrawal9221**
Updating public doc for upcoming deprecation for contact migration

### CODE REVIEW

File: skype/skype-ps/skype/Move-CsUser.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ When moving a user to the Microsoft 365 cloud to become TeamsOnly (or the revers
...
 > - To use Multi-Factor Authentication (MFA) with Move-CsUser requires either Skype for Business Server 2015 CU12 or any version of Skype for Business Server 2019. When using MFA do not specify the -Credential paremeter. If you are using an earlier version of Skype for Business Server, you should either disable MFA and use the credential parameter, or obtain a newer version of the administrative tools for Skype for Business Server that supports MFA.
 
+> [!NOTE]
+>
+> - From November 10, 2023 onwards, moving users from Teams to On-Premises will discontinue migrating their contacts. This is mainly due to Microsoft's continuous efforts to tighten the security and protect customer's data. After carefully analyzing the usage patterns and performing risk assessment with the legacy infrastructure, Microsoft decided to deprecate this feature.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Move-CsUser.md:63]:
```suggestion
> As of November 10, 2023, moving users from Teams to an on-premises deployment will no longer migrate their contacts. This is mainly due to our continuous efforts to tighten security and protect customers' data. After carefully analyzing the usage patterns and performing risk assessments with the legacy infrastructure, we decided to deprecate this feature. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ruchiagrawal9221, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by ruchiagrawal9221** [comment]
@msbemba  could you pls complete the review on this PR so this can be completed?

**Review by msbemba — APPROVED** [review]
Approved

**Comment by ruchiagrawal9221** [comment]
Can someone please merge this PR ?

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by ruchiagrawal9221** [comment]
Thanks @yogkumgit

**Pull request merged**